### PR TITLE
Improve (Open-)Harmony OS build instructions

### PR DESCRIPTION
**Pull request by jschwe**
Clarify how to build for HarmonyOS, improve the documentation of the environment varibales and mention `direnv` as a convenient tool to automatically set the environment variables.

### CODE REVIEW

File: src/hacking/building-for-openharmony.md [hunk 1]
```diff
@@ -155,33 +155,49 @@ The value of the variable must be a file path to a valid `.json` file with the s
...
-./mach build --ohos --release
+./mach build --ohos --release [--flavor=harmonyos]
 ```
 
 In `mach build`, `mach install` and `mach package` commands, `--ohos` is an alias for `--target aarch64-unknown-linux-ohos`.
-To build for an emulator running on an x86-64 host, use `--target x86_64-unknown-linux-ohos`
+To build for an emulator running on an x86-64 host, use `--target x86_64-unknown-linux-ohos`.
+The default ohos build / package / install targets OpenHarmony.
+If you want to build for HarmonyOS you can add `--flavor=harmonyos`.
+Please check the [Signing configuration](#signing-configuration) and a configuration with `"name": "hos"` and `"type": "HarmonyOS""` and the respective signing certificates.
```
Comment (delan) [inline src/hacking/building-for-openharmony.md:197]: > and a configuration with […]

missing verb?

**Review by delan — COMMENTED** [review]
I’ve pushed some minor edits, aside from that I only had one question:

File: src/hacking/building-for-openharmony.md (hunk 1 above)
Comment (jschwe) [reply src/hacking/building-for-openharmony.md]: indeed - updated.

**Comment by jschwe** [comment]
Thanks for the edits!

**Pull request merged**